apps/hash-frontend/src/pages/notes.page/timestamp-column.tsx

```diff
@@ -0,0 +1,57 @@
+import { IconButton } from "@hashintel/design-system";
+import { Box } from "@mui/material";
+import { FunctionComponent } from "react";
+
+import { ArrowsFromLineRegularIcon } from "../../shared/icons/arrows-from-line-regular-icon";
+import { ArrowsToLineRegularIcon } from "../../shared/icons/arrows-to-line-regular-icon";
+import { ChevronDownRegularIcon } from "../../shared/icons/chevron-down-regular-icon";
+import { ChevronUpRegularIcon } from "../../shared/icons/chevron-up-regular-icon";
+import { TimestampCollectionHeading } from "./timestamp-collection-heading";
+import { TimestampCollectionSubheading } from "./timestamp-collection-subheading";
+
+const timestampColumnWidth = 150;
+
+export const TimestampColumn: FunctionComponent<{
+  heading?: string;
+  subheading: string;
+  navigateUp?: () => void;
+  navigateDown?: () => void;
+  isCollapsed: boolean;
+  toggleIsCollapsed: () => void;
+}> = ({
+  heading,
+  subheading,
+  navigateUp,
+  navigateDown,
+  isCollapsed,
+  toggleIsCollapsed,
+}) => (
+  <Box
+    sx={{
+      width: timestampColumnWidth,
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "flex-end",
+    }}
+  >
+    {heading ? (
+      <TimestampCollectionHeading>{heading}</TimestampCollectionHeading>
+    ) : null}
+    <TimestampCollectionSubheading>{subheading}</TimestampCollectionSubheading>
+    <Box display="flex" marginRight={-1}>
+      <IconButton disabled={!navigateUp} onClick={navigateUp}>
+        <ChevronUpRegularIcon />
+      </IconButton>
+      <IconButton disabled={!navigateDown} onClick={navigateDown}>
+        <ChevronDownRegularIcon />
```

Comment:
This is enabled for me even without any other day to travel to

Reply:
Fixed in https://github.com/hashintel/hash/pull/3434/commits/b53ffb23c63d1036b08c689daca412d202778704